src/components/utils/OffchainUtils.ts

```diff
@@ -147,6 +146,18 @@ export const getNonce = async (account: string) => {
   }
 }
 
+export const getSessionKey = async (account: string) => {
+  try {
+    const res = await axios.post(getOffchainUrl(`/accounts/getSessionKey?account=${account}`))
```

Comment:
What does this method do? Not clear.
Main account can have multiple session keys.
Does this method check if a gived session key is an existing (known) session key in our DB? if yes, then the method and API endpoint should be called `isKnownSessionKey` and return `boolean`